src/search/identity_search.rs

```diff
@@ -57,10 +58,10 @@ pub fn identity_search(
     Ok(None)
 }
 
-fn build_query(
+pub fn build_query(
```

Comment:
Why did this need to be made `pub`? I see two functions with this same name, are we referencing the same name from different modules? Could indicate we need some more descriptive name.

Reply:
We are using the identity-related build_query method in the delete endpoint (i.e. so we can build an efficient and but specific query to delete a single compound doc). 

That said, totally makes sense to rename these methods at this point as we have a couple of different "build_query" methods.